exporter/exporter.go

```diff
@@ -116,10 +118,19 @@ func (e *Exporter) Attach() error {
 			return fmt.Errorf("multiple configs with name %q", cfg.Name)
 		}
 
-		module, err := libbpfgo.NewModuleFromFileArgs(libbpfgo.NewModuleArgs{
+		newModuleArgs := libbpfgo.NewModuleArgs{
 			BPFObjPath:      cfg.BPFPath,
 			SkipMemlockBump: true, // Let libbpf itself decide whether it is needed
-		})
+		}
+
+		if e.btfFilePath != "" {
+			if _, err := os.Stat(e.btfFilePath); err != nil {
```

Comment:
`err` might not be `file not exists`, there's a pattern to use here:

* https://stackoverflow.com/a/12518877

Reply:
Updated.